### PR TITLE
Benchmark starlight

### PR DESCRIPTION
**Pull request by girazoki**
Start adding benchmarks to starlight

### CODE REVIEW

File: solo-chains/runtime/starlight/src/weights/frame_benchmarking::baseline.rs
Comment (tmpolaczyk) [inline solo-chains/runtime/starlight/src/weights/frame_benchmarking::baseline.rs:1]: Amazing that this is a valid filename, is it expected?

```
solo-chains/runtime/starlight/src/weights/frame_benchmarking::baseline.rs
```

Comment (girazoki) [reply solo-chains/runtime/starlight/src/weights/frame_benchmarking::baseline.rs:1]: it generated it, let me remove it

File: solo-chains/runtime/dancelight/src/weights/pallet_babe.rs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use sp_std::marker::PhantomData;
+
+/// Weights for pallet_babe using the Substrate node and recommended hardware.
+pub struct SubstrateWeight<T>(PhantomData<T>);
```
Comment (tmpolaczyk) [inline solo-chains/runtime/dancelight/src/weights/pallet_babe.rs:55]: This is not used

Comment (girazoki) [reply solo-chains/runtime/dancelight/src/weights/pallet_babe.rs]: removed

File: solo-chains/runtime/dancelight/src/weights/pallet_xcm_benchmarks_fungible.rs [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use sp_std::marker::PhantomData;
+
+/// Weights for pallet_xcm_benchmarks::fungible using the Substrate node and recommended hardware.
+pub struct WeightInfo<T>(PhantomData<T>);
```
Comment (tmpolaczyk) [inline solo-chains/runtime/dancelight/src/weights/pallet_xcm_benchmarks_fungible.rs:55]: This is not used

File: solo-chains/runtime/dancelight/src/weights/pallet_xcm_benchmarks_generic.rs [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use sp_std::marker::PhantomData;
+
+/// Weights for pallet_xcm_benchmarks::generic using the Substrate node and recommended hardware.
+pub struct WeightInfo<T>(PhantomData<T>);
```
Comment (tmpolaczyk) [inline solo-chains/runtime/dancelight/src/weights/pallet_xcm_benchmarks_generic.rs:55]: This is not used

**Review by tmpolaczyk — APPROVED** [review]

File: solo-chains/runtime/dancelight/src/weights/pallet_xcm_benchmarks_generic.rs (hunk 3 above)
Comment (girazoki) [reply solo-chains/runtime/dancelight/src/weights/pallet_xcm_benchmarks_generic.rs]: fixed

File: solo-chains/runtime/dancelight/src/weights/pallet_xcm_benchmarks_fungible.rs (hunk 2 above)
Comment (girazoki) [reply solo-chains/runtime/dancelight/src/weights/pallet_xcm_benchmarks_fungible.rs]: fixed

**Pull request merged**